server/helpers/auth.js

```diff
@@ -0,0 +1,8 @@
+import jwt from 'jsonwebtoken';
+
+const makeToken = (payload) => {
```

Comment:
Nice work @GerrardE This can be refactored like this 👇 
```suggestion
const makeToken = (payload) => jwt.sign({ payload }, process.env.SECRET_KEY);
```